tests/test_end_to_end.py

```diff
@@ -63,3 +67,11 @@ def test_view_exception(self, client):
         # Asserts: Requests in progress
         assert 'starlette_requests_in_progress{method="GET",path="/bar/"} 0.0' in metrics_text
         assert 'starlette_requests_in_progress{method="GET",path="/metrics/"} 1.0' in metrics_text
+
+    def test_path_substituion(self, client):
+        client.get("/foo/baz")
+
+        response = client.get("/metrics/")
+        metrics_text = response.content.decode()
+
+        assert 'starlette_requests_total{method="GET",path="/foo/{bar}/"} 1.0' in metrics_text
```

Comment:
Is the test updated to reflect `path_template` changes ?

Reply:
Oh... totally forgot. I updated it now. 
Sorry for not answering so long!